### PR TITLE
Flask to fastapi

### PR DESCRIPTION
**Pull request by aliaminibagh**
Hi,
The last chnages has be done.

### CODE REVIEW

File: backbone/app.py [hunk 1]
```diff
@@ -3,39 +3,27 @@
...
-    model_path = request.form.get("model_path")
-    detector.set_model(model_path)
-    return "Model set"
+    contents = await image.read()
+    img = Image.open(io.BytesIO(contents))
+    results = detector.detect(img)
+    return results.to_json(orient="records")
 
 
 if __name__ == "__main__":
```
Comment (a-sharifi) [inline backbone/app.py]: Thank you for your hard-working please remove this line and with the help of the following code run unicorn directly.
https://github.com/virasad/semantic_segmentation_service/blob/b9cb50a9dc2802b206bfc262dd2d23216139156a/docker-compose.yml#L29

File: backbone/detect.py [hunk 2]
```diff
@@ -5,11 +5,8 @@
 class Detector():
     def __init__(self):
         self.model = torch.hub.load('ultralytics/yolov5', 'custom',
-                               path=os.environ.get('MODEL_PATH', 'weights/best.pt'))
-    def set_model(self, model_path):
-        self.model = torch.hub.load('ultralytics/yolov5', 'custom',
-                               path=model_path)
-    
+                                    path=os.environ.get('MODEL_PATH', 'weights/best.pt'))
```
Comment (a-sharifi) [inline backbone/detect.py]: why do change it? it should get a dynamic path. maybe someone want to change the weight, Its not rational

File: backbone/test_req.py [hunk 3]
```diff
@@ -19,6 +19,7 @@ def send_cv2_image(cv2image, detection_url):
     img_bytes = cv2.imencode('.jpg', cv2image)[1].tobytes()
     response = requests.post(detection_url, files={"image": img_bytes}).json()
     pprint.pprint(response)
```
Comment (a-sharifi) [inline backbone/test_req.py]: remove pprint here

File: train/main_train.py [hunk 4]
```diff
@@ -48,15 +47,15 @@ def train_model():
     # make raw Label Folder
     os.makedirs(os.path.join('tmp/DATASET/raw_annotations/'))
     image_path = normpath(image_path)
-
+    print(data)
```
Comment (a-sharifi) [inline train/main_train.py]: remove this

File: train/main_train.py [hunk 5]
```diff
@@ -175,5 +140,4 @@ def train_model():
 
 
 if __name__ == "__main__":
-    from waitress import serve
-    serve(app, host="0.0.0.0", port=os.environ.get("PORT", 8000))
+    uvicorn.run(app, host="0.0.0.0", port=8000)
```
Comment (a-sharifi) [inline train/main_train.py]: remove this

**Review by a-sharifi — CHANGES_REQUESTED** [review]

File: backbone/detect.py (hunk 2 above)
Comment (aliaminibagh) [reply backbone/detect.py]: yes. I forget to paste it when have test fastapi version.

**Review by a-sharifi — APPROVED** [review]

**Pull request merged**